GCMessengerSDKSample/GCMessengerSDKSample/AccountDetailsViewController.swift

```diff
@@ -168,4 +169,18 @@ extension AccountDetailsViewController: AuthenticationViewControllerDelegate {
             self.showErrorAlert(message: message)
         })
     }
+    
+    func onClientResponseError(message: String) {
+        dismiss(animated: true, completion: {
+            let alert = UIAlertController(title: "Error occurred", message: message, preferredStyle: .alert)
+            
+            alert.addAction(UIAlertAction(title: "OK", style: .cancel, handler: { _ in
+                self.loginButton.isEnabled = true
+            }))
+            
+            if let topViewController = UIApplication.getTopViewController() {
+                topViewController.present(alert, animated: true)
```

Comment:
Just to validate so we won't be repeating our past mistakes, is it required to have a 'ui blocker' view? As this view will limit the e2e capabilities.